doc/examples/pilz_industrial_motion_planner/pilz_industrial_motion_planner.rst

```diff
@@ -95,7 +93,7 @@ phases as the lead axis.
 Input parameters in ``moveit_msgs::MotionPlanRequest``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- ``planner_id``: PTP
+- ``planner_id``: ``"PTP"``
 - ``group_name``: name of the planning group
```

Comment:
```suggestion
- ``group_name``: the name of the planning group
```

This applies to the entire list and the next list, excluding line 117 (``trajectory/joint_trajectory/joint_names``: a list of the joint names of the generated joint trajectory) which does include the article

Reply:
This also applies for the lists in LIN and CIRC